packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -33,6 +33,8 @@ import { cloneDeep, isNumber } from "lodash-es"
 import { getStatusCodeReasonPhrase } from "~/helpers/utils/statusCodes"
 
 export const OPENAPI_DEREF_ERROR = "openapi/deref_error" as const
+export const OPENAPI_UNRESOLVED_REFS_ERROR =
+  "openapi/unresolved_refs_error" as const
```

Comment:
This is not referenced anywhere.

Reply:
It was in the previous integration. I have removed it now.